applications/registration-react/src/pages/dataset-registration-page/form-concept/concept-tag-input-field/concept-tags-input-field.component.jsx

```diff
@@ -179,7 +167,7 @@ class InputTagsFieldConcepts extends React.Component {
           }}
           onSuggestionsClearRequested={this.onSuggestionsClearRequested}
           onSuggestionsFetchRequested={this.onSuggestionsFetchRequested}
-          renderInputComponent={autosizingRenderInput}
+          renderInputComponent={props => <AutosizeInput {...props} />}
```

Comment:
Declaring render function this way may have a significant impact on performance.

Reply:
jeg tror jeg prøvde den andre måte, men husker ikke hva var feil... må prøve igjen da

Reply:
fjernet hele greia